include/date_time.h

```diff
@@ -104,6 +119,17 @@ int date_time_now(int64_t *unix_time_ms);
  */
 bool date_time_is_valid(void);
 
+/** @brief Convenience function that checks if the library has obtained
+ *	   an initial valid date time and timezone.
+ *
+ *  @note If this function returns false there is no point of
+ *	  subsequent calls to date_time_now_local().
+ *
+ *  @return true  The library has obtained a local date time.
+ *  @return false The library has not obtained a local date time.
+ */
+bool date_time_is_valid_local(void);
```

Comment:
Are you using this function at the moment without needing the time value? If not, I wouldn't add this function to the API until there is a need for it.

Reply:
Ok, that's fine then. Apart from the unit test comment, I don't think I have anything more.

Reply:
Unfortunately I use the SDK out of tree and don't have the unit test framework set up. Feel free to close the PR if that's a deal-breaker.

Reply:
Ok, I'll add the test later.